### ISSUE TITLE
Wrong ordering in ordered list

### ISSUE THREAD

**Issue by rageshkrishna** [Gioni06/terminal.css#1]
What it says -- the ordering on ordered lists is wrong. I see 1, 3, 2, 3 instead of 1, 2, 3, 4. I think the nested list might be throwing something off:

![capture](https://user-images.githubusercontent.com/1134972/51785016-89e7b780-2177-11e9-9ae0-d2de9129a629.PNG)

**Comment by Gioni06** [comment]
Good catch. I'll take a look at it. Thanks for reporting! BTW you are free to contribute anytime to this project :) I'm actively working on it and there are a lot of things missing as of today.

**Comment by Gioni06** [comment]
Fixed

**Comment by Gioni06** [comment]
Apparently, the changes didn't quite fix the issue. I'll take another look.
